test/DebugInfo/NonSemantic/DebugSourceContinued.ll

```diff
@@ -0,0 +1,46 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc --spirv-debug-info-version=nonsemantic-shader-100 -spirv-text -o %t.spt
+; RUN: FileCheck %s --input-file %t.spt --check-prefix CHECK-SPIRV
+; RUN: llvm-spirv %t.bc  --spirv-debug-info-version=nonsemantic-shader-100 -o %t.spv
+; RUN: llvm-spirv -r -emit-opaque-pointers %t.spv -o %t.rev.bc
+; RUN: llvm-dis %t.rev.bc -o %t.rev.ll
+; RUN: FileCheck %s --input-file %t.rev.ll --check-prefix CHECK-LLVM
+
+; CHECK-LLVM: !DIFile(filename: "t.c", directory: "/test", source: "A
+; CHECK-LLVM-COUNT-600000: A
+
+; CHECK-SPIRV: String [[#Str:]] "A
+; CHECK-SPIRV-COUNT-262130: A
```

Comment:
COUNT is something new for me. Naïve calculation 262130 * 2 + 75737 = 599997 != 600000
Does COUNT counts from 1 or from 0? I'm trying to find missing "AAA".

Reply:
You can see that the "first A" is checked in the line above. So, somehow it is already counted, and the correct calculation will be: `(262130 + 1) * 2 + (75737 + 1) = 600000`.

For LLVM checks however it does the counting "once again" and `600000` is a correct check